### DISCUSSION TITLE
running on Raspi? ESP fails all time

### DISCUSSION THREAD

**Discussion by Starter4711** [shining-man/bsc_fw#203] [General]
Hello!
may I ask, if it is possible to run BSC on an Raspi? 
The ESP always fails on connecting to BT or Wlan. (Neey Heltec 4A Balancer)
Thx

**Comment by tueftler0** [comment]
Hi. No, there is no possibility to install it on a RaspberryPi.
We are aware of the problem and there will be a solution for it in the near future.
It is mainly caused by the built-in antenna constantly having to switch between WiFi and BT.
When there is a ready solution, we will announce it in our BSC-Discord-Channel at first.
